src/main/java/com/pinterest/secor/uploader/S3UploadManager.java

```diff
@@ -128,6 +128,9 @@ public void refresh() {}
         if (mConfig.getAwsClientPathStyleAccess()) {
             S3ClientOptions clientOptions = new S3ClientOptions();
             clientOptions.setPathStyleAccess(true);
+            if (System.getenv("SECOR_DEV") == "true"){
+                clientOptions.disableChunkedEncoding();
```

Comment:
What was the problem with checked encoding?

Reply:
And when is SECOR_DEV being set?

Reply:
I guess that @nikshepsvn are setting `SECOR_DEV` only in his development environment. IMHO this approach are like "easter eggs", better approach are use properties to change this, where it must be well documented what is the propose of this setting.